src/brevitas/graph/calibrate.py

```diff
@@ -85,42 +101,49 @@ def disable_act_quant_hook(self, module, inp, output):
                 inp, min_val=module.quant_injector.min_val, max_val=module.quant_injector.max_val)
         return QuantTensor(value=inp, training=module.training)
     
-    def disable_act_quantization(self, model):
+    def disable_act_quantization(self, model, is_training):
         for module in model.modules():
             if isinstance(module, ActQuantProxyFromInjector):
                 hook = module.register_forward_hook(self.disable_act_quant_hook)
+                module.train(is_training)
                 self.disable_act_quant_hooks.append(hook)
             elif isinstance(module, _ACC_PROXIES):
+                module.train(is_training)
                 module.disable_quant = True
 
-    def disable_param_quantization(self, model):
+    def disable_param_quantization(self, model, is_training):
         for module in model.modules():
             if isinstance(module, _PARAM_PROXIES):
+                module.train(is_training)
                 module.disable_quant = True
     
-    def enable_act_quantization(self, model):
+    def enable_act_quantization(self, model, is_training):
         for module in model.modules():
             if isinstance(module, _ACC_PROXIES):
+                module.train(is_training)
                 module.disable_quant = False
+            elif isinstance(module, ActQuantProxyFromInjector):
+                module.train(is_training)
         for hook in self.disable_act_quant_hooks:
             hook.remove()
         self.disable_act_quant_hooks = []
 
-    def enable_param_quantization(self, model):
+    def enable_param_quantization(self, model, is_training):
         for module in model.modules():
             if isinstance(module, _PARAM_PROXIES):
                 module.disable_quant = False
+                module.train(is_training)
 
 
 class DisableQuantInference(DisableEnableQuantization):
```

Comment:
We don't need a DisableQuantInference class anymore as inference is performed externally now. The apply method can be moved to DisableEnableQuantization and BiasCorrection can inherit that and then override it.

Reply:
Ok. Maybe in #460 we could move away from the "apply" method and simply use "__call__".

Reply:
let's do it in a separate PR over all graph transformations, it's more work